src/altinn-app-frontend/src/utils/render.tsx

```diff
@@ -74,14 +70,14 @@ export function renderValidationMessages(
   }
 
   return (
-    <MessageComponent
-      messageType='error'
-      style={messageComponentStyle}
-      key='error'
-      id={id}
-    >
-      <ol>{messages.map(validationMessagesToList)}</ol>
-    </MessageComponent>
+    <div style={{ paddingTop: '0.375rem' }}>
+      <ErrorMessage
+        id={id}
+        key='error'
```

Comment:
Key should be removed here, ref docs about how they are used https://reactjs.org/docs/lists-and-keys.html#keys